pkg/reconciler/layer3.go

```diff
@@ -123,6 +175,24 @@ func (r *reconcile) createVrfConfigMap(l3vnis []networkv1alpha1.VRFRouteConfigur
 	return vrfConfigMap, nil
 }
 
+func createVrfFromTaaS(taas []networkv1alpha1.RoutingTable) map[string]frr.VRFConfiguration {
+	vrfConfigMap := map[string]frr.VRFConfiguration{}
+
+	for i := range taas {
+		spec := taas[i].Spec
+
+		name := fmt.Sprintf("taas.%d", spec.TableID)
```

Comment:
this could break hardly in BM4X if we don't extend the interface naming in linux.

Reply:
Ahh I just miscalculated it ^^